src/SystemServicesContext/useIsLocationOn.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState, useCallback } from "react"
+import { Platform } from "react-native"
+
+import useOnAppStateChange from "./useOnAppStateChange"
+import { isLocationEnabled } from "../gaen/nativeModule"
+
+const useIsLocationOn = (): boolean => {
+  const [isLocationOn, setIsLocationOn] = useState(false)
+
+  const determineIsLocationOn = useCallback(async () => {
+    if (Platform.OS === "android") {
+      isLocationEnabled().then((result) => setIsLocationOn(result))
+    } else {
+      setIsLocationOn(true)
```

Comment:
this `if the platform is ios, then set the locationOn to be true` jumps out as odd to me as we dont actually know the location permission status for ios (and we know that we _cant_ be asking for location permission on ios anyways).  Further, we know that we dont need to check if location is on if we know that the location status is not required, so these are pretty tightly linked. what do you think about combining these hooks into one and having them return a tuple or object?

let me know if you would like to discuss irl!